app/helpers/format-profile.js

```diff
@@ -0,0 +1,14 @@
+import {helper} from '@ember/component/helper'
+
+const hrefRegex = /href="(.*?)"/gm
+const typeAndIdRegex = /([a-z]+(?:\/))([0-9]+(?:$|(?=[?-])|(?=\/$)))/gm
```

Comment:
`typeAndIdFromUrl`?

Reply:
Would also make a nice little `npm` package no? Could be reused for sure in r4

Reply:
for npm package i'd say let's wait a bit?

here's a very basic test https://github.com/internet4000/explorer-discogs/blob/086d53cc2fc0c7726082ddc8835f7fda803f884d/tests/integration/helpers/format-profile-test.js